sdk/scheduler/src/test/java/com/mesosphere/sdk/offer/evaluate/OfferEvaluatorTest.java

```diff
@@ -60,7 +60,7 @@ public void testReserveLaunchScalar() throws Exception {
         Assert.assertEquals(getResourceId(reserveResource), getResourceId(launchResource));
         String executorId = launchOperation.getLaunchGroup().getExecutor().getExecutorId().getValue();
 
-        String prefix = TestConstants.POD_TYPE + CommonIdUtils.NAME_ID_DELIM;
+        String prefix = TestConstants.POD_TYPE + "__";
```

Comment:
Any reason to not keep the delimiter as public?

Reply:
This is kind of an issue with the Utils idiom, IMO, since it doesn't actually encapsulate the data it operates on. That won't change here for a while, though, so I vote for public too.